src/controller/artifact/controller.go

```diff
@@ -118,6 +121,8 @@ type Controller interface {
 	Walk(ctx context.Context, root *Artifact, walkFn func(*Artifact) error, option *Option) error
 	// HasUnscannableLayer check artifact with digest if has unscannable layer
 	HasUnscannableLayer(ctx context.Context, dgst string) (bool, error)
+	// List accessory records by artifact id if this artifact is a accessory
+	ListAccByArtID(ctx context.Context, ArtifactID int64) (accs []accessorymodel.Accessory, err error)
```

Comment:
the function sounds like to get the accessories that belongs to the subject artifacts, and why it returns an array? instead of a pointer of the accessory object.

Reply:
It would be more flexible for further function reference.